apps/web/lib/getTeamMemberEmailFromCrm.ts

```diff
@@ -19,23 +19,23 @@ interface EventData {
   schedulingType: SchedulingType | null;
   metadata: Prisma.JsonValue | null;
   length: number;
+  hosts: {
+    user: {
+      email: string;
+    };
+  }[];
 }
 
 const returnNullValue = { email: null, recordType: null, crmAppSlug: null };
 
 async function findUserByEmailWhoIsAHostOfEventType({
   email,
-  eventTypeId,
+  eventData,
 }: {
   email: string;
-  eventTypeId: number;
+  eventData: EventData;
 }) {
-  return prisma.user.findFirst({
-    where: {
-      email,
-      hosts: { some: { eventTypeId } },
-    },
-  });
+  return !!eventData.hosts.find((host) => host.user.email === email);
```

Comment:
This wouldn't work now as in apps/web/lib/team/[slug]/[type]/getServerSideProps.tsx file we limit the number of hosts to fetch

Reply:
and if we fetch all the hosts then breaks the OG image and page load would be much slower because it takes 1-3 seconds more to fetch the hosts for some customers